packages/amplify-e2e-core/src/index.ts

```diff
@@ -25,9 +25,16 @@ declare global {
 const amplifyTestsDir = 'amplify-e2e-tests';
 
 export function getCLIPath(testingWithLatestCodebase = false) {
-  return testingWithLatestCodebase
-    ? path.join(__dirname, '..', '..', 'amplify-cli', 'bin', 'amplify')
-    : process.env.AMPLIFY_PATH || 'amplify';
+  if (testingWithLatestCodebase) {
+    return process.env.AMPLIFY_PATH || (process.platform === 'win32' ? 'amplify.exe' : 'amplify');
+  }
+
+  const amplifyScriptPath = path.join(__dirname, '..', '..', 'amplify-cli', 'bin', 'amplify');
+  return amplifyScriptPath;
```

Comment:
this logic seems to handle `testingWithLatestCodebase` opposite from what it was prior to the change. Is that intentional?